### PR TITLE
feat(experimental): `BadgedContent` add new component

### PR DESCRIPTION
**Pull request by vladimirpotekhin**
## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring
- [ ] Build or CI related changes
- [ ] Tests related changes
- [ ] Documentation content changes

## What is the current behavior?

Closes #

### CODE REVIEW

**Review by splincode — APPROVED** [review]

File: projects/demo/src/modules/experimental/badged-content/badged-content.component.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    readonly example2: TuiDocExample = {
+        HTML: import('./examples/2/index.html?raw'),
+    };
+
+    readonly example3: TuiDocExample = {
+        HTML: import('./examples/3/index.html?raw'),
+    };
+
+    radiusVariants = ['0.75rem', '50%'];
```
Comment (waterplea) [inline projects/demo/src/modules/experimental/badged-content/badged-content.component.ts:31]: Can it be `var(--tui-radius-l)` by default?

File: projects/demo/src/modules/experimental/badged-content/examples/import/import-module.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+```ts
+import {NgModule} from '@angular/core';
+import {TuiBadgeAlertModule} from '@taiga-ui/experimental';
+// ...
+
+@NgModule({
+  imports: [
+    // ...
+    TuiBadgeAlertModule,
```
Comment (waterplea) [inline projects/demo/src/modules/experimental/badged-content/examples/import/import-module.md:9]: Wrong examples

File: projects/experimental/components/badged-content/badged-content.style.less [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    }
+
+    &_bottom {
+        bottom: var(--t-offset);
+        right: var(--t-offset);
+    }
+}
+
+.t-border:not(:empty) {
+    --tui-duration: 0;
```
Comment (waterplea) [inline projects/experimental/components/badged-content/badged-content.style.less:31]: What do we need it for?

File: projects/experimental/components/badged-content/badged-content.style.less [hunk 4]
```diff
@@ -0,0 +1,35 @@
+@import 'taiga-ui-local';
+
+:host {
+    --t-badge-height: 0;
+    --t-radius: 0.75rem;
```
Comment (waterplea) [inline projects/experimental/components/badged-content/badged-content.style.less:5]: `--tui-radius`? If it's a public API var. Also it looks like it should default to `var(--tui-radius-l)`.

**Review by waterplea — APPROVED** [review]

File: projects/experimental/components/badged-content/badged-content.style.less (hunk 3 above)
Comment (vladimirpotekhin) [reply projects/experimental/components/badged-content/badged-content.style.less:31]: This is to remove the initial background transition animation for regular (non-alert) badges.

Comment (vladimirpotekhin) [reply projects/experimental/components/badged-content/badged-content.style.less]: hmm, really)
I left the transition only for hoverable badges 👍

**Pull request merged**